### PR TITLE
Fix Denis's comments regarding MaaS documentation

### PR DESCRIPTION
**Pull request by ImenBizid**

### CODE REVIEW

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 1]
```diff
@@ -331,11 +331,21 @@ The following workflows represent some mathematical functions that can be optimi
 
 *Himmelblau_Function:* is a multi-modal function, used to test the performance of optimization algorithms.
 
-== Model as a Service
+== Model as a Service (MaaS)
 
-Using MLOS, you can deploy any machine learning model as a web service in any compute resource deployed on Proactive.
+Once a predictive model is built, tested and validated, you can easily use it in real world production pipelines by deploying it as a Model-Service.
```
Comment (mboussaa) [inline src/docs/MLOS/MLOSUserGuide.adoc]: by deploying it as a service

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 2]
```diff
@@ -331,11 +331,21 @@ The following workflows represent some mathematical functions that can be optimi
...
+This will be particularly useful for engineering teams that want to take advantage of this model.
 
-The following deployment steps are similar wherever you need to deploy your model:
+Interfacing with the Web Service could be setup in different way:
```
Comment (mboussaa) [inline src/docs/MLOS/MLOSUserGuide.adoc]: ways

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 3]
```diff
@@ -331,11 +331,21 @@ The following workflows represent some mathematical functions that can be optimi
...
+This will be particularly useful for engineering teams that want to take advantage of this model.
 
-The following deployment steps are similar wherever you need to deploy your model:
+Interfacing with the Web Service could be set up in different ways:
+
+- Either providing a user identifier deploy the model of your choice and having the Web Service available to compute the prediction and return its value.
+
+- Or by accepting a payload, converting it to json data, making the prediction using MaaS and returning its value.
+
+Using MaaS, you can easily deploy and use any machine learning model as a Web Service in any compute host, physical or virtual, on which there is a Proactive Node available.
```
Comment (mklkun) [inline src/docs/MLOS/MLOSUserGuide.adoc]: compute host => computing host

Comment (mboussaa) [inline src/docs/MLOS/MLOSUserGuide.adoc]: on physical or virtual compute host on which there is a Proactive Node available.

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 4]
```diff
@@ -331,11 +331,21 @@ The following workflows represent some mathematical functions that can be optimi
...
+You can also trigger the deployment of a specific VM using the Resource Manager elastic policies, and continue by deploying a Model-Service on that specific node.
```
Comment (mboussaa) [inline src/docs/MLOS/MLOSUserGuide.adoc]: and eventually, deploy a ...

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 5]
```diff
@@ -331,11 +331,21 @@ The following workflows represent some mathematical functions that can be optimi
...
+
+The following deployment steps are similar wherever you need to deploy and use your model:
```
[the remark this answers is not in the document]
Comment (ImenBizid) [reply src/docs/MLOS/MLOSUserGuide.adoc]: wherever referring to the compute host

**Pull request merged**